raster-foundry/raster-foundry
# Enhance dev setup ever so slightly

## Overview

This PR forwards the postgres container's port to a non-standard postgres port (so it won't be mad if you have a postgres service running on your host), removes `initialCommandsInConsole` since they were mad about unused imports, were all broken, and included things beyond what we need, and uses the same transactor in db tests as is used in the application, but + rollback. Its purpose is to allow people running sbt outside the docker zone to run tests against the postgres container.

### Checklist


### Notes

It's a pretty selfish pr but it still has some value maybe to other people, especially if we ever get language server integration working outside of IntelliJ.

## Testing Instructions

 * create a `.env.local` that points to `localhost:5434` in the POSTGRES_URL variable
 * source it
 * run `./sbt` on your host
 * try to run database tests
 * it should be fine
